CONTRIBUTING.md

```diff
@@ -87,24 +92,36 @@ When creating a new Observability Pack or reviewing a PR please keep the followi
 
 #### Author
 
-> The creator(s) of the pack
+> The creator(s) or contributors of the pack
 
 - `Required`
 - No limit to the amount of authors
 - A Github Action job verifies pack level
 
+#### Summary
+
+> A brief summary of the pack functionality.
+
+- `Required`
```

Comment:
This is not currently required in the json schema

Reply:
it is in graphql though...

<img width="345" alt="Screen Shot 2021-08-19 at 9 11 05 AM" src="https://user-images.githubusercontent.com/1395158/130093976-efd9ec93-b546-45e0-892a-14f696ae6da2.png">